src/main/java/fr/minuskube/inv/content/InventoryContents.java

```diff
@@ -104,6 +104,18 @@ public SlotIterator newIterator(SlotIterator.Type type, SlotPos startPos) {
         @Override
         public ClickableItem[][] all() { return contents; }
 
+        @Override
+        public List<SlotPos> slots() {
+            List<SlotPos> slotPos = new ArrayList<>();
+            for (int row = 0; row < contents.length; row++) {
+                for(int column = 0; column < contents[0].length; column++) {
+                    if(!this.get(row, column).isPresent())
```

Comment:
I think the method should return all the inventory slots, not only the non-empty ones, so I think you should remove this condition